sample/OSCON17/sessions

```diff
@@ -0,0 +1,548 @@
+		"slides": null,
+		"audio": null,
+		"video": null,
+		"microlocation": {
+			"id": 1,
+			"name": "Meeting Room 9 A/B"
+		},
+		"signup_url": null,
+		"speakers": [{
+			"id": 9,
+			"name": "Guru Chahal",
+			"organisation": "Avi Networks"
+		}],
+		"state": "accepted",
+		"timezone": "UTC"
+	},
+	{
+		"id": 9,
+		"title": "How can I contibute? A guide to making your first open source contribution",
+		"subtitle": null,
+		"short_abstract": null,
+		"long_abstract": "This talk is for you—the documentarians, developers, students, or community members wondering what you can contribute to open source and how to get started. Lucy Wyman discusses several ways open source projects need your help, what to look for in a project you're contributing to, and some first steps to making your first pull request.",
+		"start_time": "2017-05-10T11:00:00",
+		"end_time": "2017-05-10T11:40:00",
+		"session_type": {
+
+		},
+		"track": {
+			"id": 1,
+			"name": "Open source: From consumer to contributor"
+
+		},
+		"comments": "",
+		"language": null,
+		"slides": null,
+		"audio": null,
+		"video": null,
+		"microlocation": {
+			"id": 12,
+			"name": "Meeting Room 9 C"
+		},
+		"signup_url": null,
+		"speakers": [{
+			"id": 10,
+			"name": "Lucy Wyman",
+			"organisation": "Puppet labs"
+		}],
+		"state": "accepted",
+		"timezone": "UTC"
+	},
+	{
+		"id": 10,
+		"title": "Application security: from zero to hero",
+		"subtitle": null,
+		"short_abstract": null,
+		"long_abstract": "While the rest of the world tries to solve the problems of insecure software with firewalls and intrusion detection, Jeremy Anderson explains how to solve the problem where it starts: at the code that defines it. Join Jeremy to learn how to fix code security defects when they’re created instead of during production when it’s already too late.",
+		"start_time": "2017-05-10T11:00:00",
+		"end_time": "2017-05-10T11:40:00",
+		"session_type": {
+
+		},
+		"track": {
+			"id": 6,
+			"name": "Security sessions"
+		},
+		"comments": "",
+		"language": null,
+		"slides": null,
+		"audio": null,
+		"video": null,
+		"microlocation": {
+			"id": 2,
+			"name": "Meeting Room 10 A/B"
+		},
+		"signup_url": null,
+		"speakers": [{
+			"id": 11,
+			"name": "Jeremy Anderson",
+			"organisation": "Cambia Health solutions"
+		}],
+		"state": "accepted",
+		"timezone": "UTC"
+	},
+	{
+		"id": 11,
+		"title": "Adopting open source in your organisation",
+		"subtitle": null,
+		"short_abstract": null,
+		"long_abstract": "Recently, Microsoft went from calling open source, a cancer to being the biggest contributor on GitHub. Edward Thomson explains how Microsoft, one of the unlikeliest software vendors, began to embrace and even extend (but not extinguish) open source software and how you can begin using and contributing to open source software in your organization.",
+		"start_time": "2017-05-10T11:00:00",
+		"end_time": "2017-05-10T11:40:00",
+		"session_type": {
+
+		},
+		"track": {
```

Comment:
Add track info including color and other fields, or it will crash the app

```
{"color": "#ff3737", "font_color": "#ffffff", "id": 216, "name": "Open Tech"}
```